pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -40,7 +40,7 @@ let
   libc_lib = if libc == null then null else getLib libc;
   cc_solib = getLib cc;
   # The wrapper scripts use 'cat' and 'grep', so we may need coreutils.
-  coreutils_bin = if nativeTools then "" else getBin coreutils;
+  coreutils_bin = if (nativeTools || targetPlatform.isWindows) then "" else getBin coreutils;
```

Comment:
I still think this one is in error? It means cc-wrapper *won't* use coreutils, which seems wrong.

Reply:
But that means you won't use them *on* Linux when you are making windows binaries. Use `hostPlatform` to turn it just when compiling on Windows (for any target).

Reply:
hmm... I'll try `hostPlatform`, let's see. My issue was that it tried to buld coreutils for windows when cross compiling windows stuff on linux.